infra/ansible/roles/mediawiki/tasks/main.yml

```
@@ -96,7 +96,7 @@
   file:
     src: /etc/nginx/sites-available/mediawiki
     dest: /etc/nginx/sites-enabled/mediawiki
-    state: link
+    state: absent  # 2023-05-21: wiki is disabled
```

I try a deployment on my local VM but this command returns an error (sorry I didn't copy it).

I was able to make the deployment works with the following code instead:

```suggestion
    path: /etc/nginx/site-enabled/mediawiki
    state: absent  # 2023-05-21: wiki is disabled
```

---

Good catch , thanks.